phones/models.py

```diff
@@ -212,6 +216,9 @@ class InboundContact(models.Model):
     relay_number = models.ForeignKey(RelayNumber, on_delete=models.CASCADE)
     inbound_number = models.CharField(max_length=15)
     last_inbound_date = models.DateTimeField(default=last_inbound_date_default)
+    last_inbound_type = models.CharField(
+        max_length=4, choices=LAST_CONTACT_TYPE_CHOICES, default="text"
```

Comment:
question: any reasons on why max length 4?

Reply:
Not any good one. Since it's storing a finite set of options and they're both 4-characters long, I thought we might as well save DB storage space.